Source/BugsnagCrashReport.m

```diff
@@ -620,6 +645,42 @@ - (NSString *_Nullable)enhancedErrorMessageForThread:(NSDictionary *_Nullable)th
 
 @end
 
+@implementation FallbackReportData
```

Comment:
It'd be good to add some brief documentation on what 'h' and 'u' mean for future travellers, either here or in `BSG_KSCrashC.c`